### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by Kendiya242**
login page

### CODE REVIEW

**Review by RuthAnn1 — COMMENTED** [review]
Is the background the color we have for the figma?  I work a lot with rgb and was usure on the color call in the class form.

File: src/component/login-form/Login.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.form{
+    border-radius: 25px;
+  background: #356589;
```
Comment (dinaolmelak) [inline src/component/login-form/Login.module.scss:3]: @RuthAnn1 , Were you referring to this color?

**Review by dinaolmelak — CHANGES_REQUESTED** [review]

File: src/component/login-form/Login.module.scss (hunk 1 above)
Comment (RuthAnn1) [reply src/component/login-form/Login.module.scss:3]: yeah I was just checking to make sure it was the color we wanted.

Comment (dinaolmelak) [reply src/component/login-form/Login.module.scss:3]: Yeah, please refer to the color pallet found in the `src/assets` folder.
Screenshot 2024-07-26 at 7 25 11 in the evening

File: src/component/login-form/Loginform.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+     <button  type="submit">Sign Up</button>
+     <div className={styles.divider}></div>
+     <button  type="submit">Log In</button>
+    
+<br></br>
+     <input className={styles.forget} type="checkbox" id="Remember me" name="Remember me" value="Remember me" />
+                        <label htmlFor="Remember me"> Remember me</label>
+<br></br>
+     <input type="checkbox" id="Forget username/password" name="Forget username/password" value="Forget username/password" />
```
Comment (dinaolmelak) [inline src/component/login-form/Loginform.jsx:28]: Chan you turn this into a button?

File: src/pages/OrgLogin.jsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
+import LoginForm from "../component/login-form/Loginform.jsx";
 export default function OrganizationLogin(){
     return(
         <h1>
-            Log In Here
+            <LoginForm></LoginForm>
```
Comment (dinaolmelak) [inline src/pages/OrgLogin.jsx:5]: can use a self closing tag `<LoginForm/>`

**Review by dinaolmelak — CHANGES_REQUESTED** [review]

File: src/component/login-form/Login.module.scss (hunk 1 above)
Comment (Kendiya242) [reply src/component/login-form/Login.module.scss:3]: Noted

File: src/pages/OrgLogin.jsx
Comment (dinaolmelak) [inline src/pages/OrgLogin.jsx:1]: There is an `OrgLogin.jsx` in the `/src/pages/org-login/` directory. So I don't think this file would be necessary.

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
The `OrgLogin` Capability will be discontinued, please place the `Login` button else where.

**Comment by dinaolmelak** [comment]
@Kendiya242 I have gone ahead and fixed the changes I requested from you.

**Review by dinaolmelak — APPROVED** [review]
Looks good to me.

**Pull request merged**